src/main/java/se/sundsvall/messaging/service/mapper/RequestMapper.java

```diff
@@ -20,128 +21,128 @@
-                .withPartyIds(List.of(partyId))
-                .withExternalReferences(request.party().externalReferences())
-                .build())
-            .withContentType(request.contentType())
-            .withSubject(request.subject())
-            .withDepartment(request.department())
-            .withBody(request.body())
-            .withAttachments(request.attachments().stream()
-                .filter(LetterRequest.Attachment::isIntendedForDigitalMail)
-                .map(attachment -> DigitalMailRequest.Attachment.builder()
-                    .withFilename(attachment.filename())
-                    .withContent(attachment.content())
-                    .withContentType(attachment.contentType())
-                    .build())
-                .toList())
-            .withOrigin(request.origin())
-            .build();
-    }
-
-    public SnailMailRequest toSnailMailRequest(final LetterRequest request, final String partyId) {
-        return SnailMailRequest.builder()
-            .withParty(SnailMailRequest.Party.builder()
-                .withPartyId(partyId)
-                .build())
-            .withDepartment(request.department())
-            .withDeviation(request.deviation())
-            .withAttachments(request.attachments().stream()
-                .filter(LetterRequest.Attachment::isIntendedForSnailMail)
-                .map(attachment -> SnailMailRequest.Attachment.builder()
-                    .withName(attachment.filename())
-                    .withContent(attachment.content())
-                    .withContentType(attachment.contentType())
-                    .build())
-                .toList())
-            .withOrigin(request.origin())
-            .build();
-    }
+		defaultEmailRequestSender = EmailRequest.Sender.builder()
+			.withName(defaults.email().name())
+			.withAddress(defaults.email().address())
+			.withReplyTo(defaults.email().replyTo())
+			.build();
+	}
+
+	public String toSmsRequest(final Message message, final String mobileNumber) {
+		var originalMessage = fromJson(message.content(), MessageRequest.Message.class);
+
+		var sender = ofNullable(originalMessage.sender())
+			.map(MessageRequest.Message.Sender::sms)
+			.map(MessageRequest.Message.Sender.Sms::name)
+			.orElse(defaultSmsRequestSender);
+
+		var smsRequest = SmsRequest.builder()
+			.withParty(SmsRequest.Party.builder()
+				.withPartyId(ofNullable(originalMessage.party())
+					.map(MessageRequest.Message.Party::partyId)
+					.orElse(null))
+				.withExternalReferences(ofNullable(originalMessage.party())
+					.map(MessageRequest.Message.Party::externalReferences)
+					.orElse(null))
+				.build())
+			.withSender(sender)
+			.withMobileNumber(mobileNumber)
+			.withMessage(originalMessage.message())
+			.withOrigin(message.origin())
+			.build();
+
+		return toJson(smsRequest);
+	}
+
+	public String toEmailBatchRequest(final Message message, final String emailAddress) {
```

Comment:
Should this method really have been renamed ?

Reply:
Nope